tests/tests_unit/test_data_classes/test_data_models/test_instances.py

```diff
@@ -176,13 +176,19 @@ def test_expand_properties(
         self, node_dumped: dict[str, Any], edge_dumped: dict[str, Any], inst_cls: type[Node] | type[Edge]
     ) -> None:
         raw = node_dumped if inst_cls is Node else edge_dumped
+        raw_no_properties = raw.copy()
+        raw_no_properties["properties"] = {}
         not_expanded = inst_cls._load(raw).to_pandas(expand_properties=False)
         expanded = inst_cls._load(raw).to_pandas(expand_properties=True, remove_property_prefix=True)
         expanded_with_prefix = inst_cls._load(raw).to_pandas(expand_properties=True, remove_property_prefix=False)
+        expanded_with_empty_properties = inst_cls._load(raw_no_properties).to_pandas(
+            expand_properties=True, remove_property_prefix=True
+        )
```

Comment:
Let's make a separate test for "test when properties are missing". Also add one for the list classes that are currently missing.